### DISCUSSION TITLE
How to force refresh DOM without cy.refresh()?

### DISCUSSION THREAD

**Discussion by MrYogurt** [cypress-io/cypress#17842] [Component Testing]
I have 2 iframes and after manipulation with the first frame, changes occur in the second. But cypress doesn't seeing it, because DOM is cached when site loaded. How i can force update this cache without cy.refresh()?

Upd: that's my fault, can be closed.

**Comment by JessicaSachs** [comment]
> I have 2 iframes and after manipulation with the first frame, changes occur in the second. But cypress doesn't seeing it, because DOM is cached when site loaded. How i can force update this cache without cy.refresh()?
> 
> 
> 
> Upd: that's my fault, can be closed.

Can you post the answer and mark it accepted, please?
